core/roboconf-target-iaas-occi/src/main/java/net/roboconf/target/occi/internal/OcciVMUtils.java

```diff
@@ -347,9 +382,12 @@ public static String createCloudAutomationVM(String hostIpPort, String image, St
 						+ "\"occi.compute.userdata\": \"" + userDataScript + "\",\n"
 						+ "\"occi.category.title\": \"scriptMixin\"\n"
 					+ "}\n}\n}";
+
+			final Logger logger = Logger.getLogger( OcciVMUtils.class.getName());
 			logger.finest(request);
-			httpURLConnection.setRequestProperty("Content-Length", "" +
-					Integer.toString(request.getBytes().length));
+			httpURLConnection.setRequestProperty(
```

Comment:
It used to work on your machine (and mine).
But Findbugs issues a warning as the output depends on the OS. To have something that is always predicatable, it is better to force UTF-8 encoding when converting bytes to strings. It is a good habit.